<launch>
    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="default_trig_val" type="bool" value="true" />
        <!-- <param name="dev" type="str" value="/dev/input/js0" /> -->
    </node>
    
    <param name="pulseRate" type="int" value="1000" />
    <param name="cmd_topic" type="str" value="/husky_velocity_controller/cmd_vel" />
    <param name="max_lin_vel" type="double" value="0.2" />
    <param name="max_rot_vel" type="double" value="0.2" />

    <node pkg="steering" type="Circadian.py" name="circadian" />
    <node pkg="steering" type="Steering.py" name="steering" />
</launch>